content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -30,18 +30,283 @@ By enabling Mobile Session Replay, you can automatically mask sensitive elements
 
 ## Configuring masking modes
 
-Using the masking modes below, you can override the default setup on a per-application basis.
+## Fine-Grained Masking
+Using the masking modes below, you can override the default setup on a per-application basis. Masking is fine-grained — you can override masking for text and inputs, images, and touches individually to create a custom configuration that suits your needs. 
```

Comment:
```suggestion
Using the masking modes below, you can override the default setup on a per-application basis. Masking is fine-grained, which means you can override masking for text, text inputs, images, and touches individually to create a custom configuration that suits your needs. 
```

Reply:
additionally, the privacy setting for text and inputs is a single setting so dividing them here may be confusing to customers who would expect separate apis for each one.

Reply:
@mariedm and @jonathanmos thank you both for the clarification. I've changed my suggestion back to the original text but kept "which means"